app/controllers/QuestionnaireController.scala

```diff
@@ -51,13 +53,13 @@ class QuestionnaireController @Inject()(val connector: DefaultAuditConnector,
   }
 
   def feedbackThankyou: Action[AnyContent] = Action.async { implicit request =>
-      Future.successful(Ok(views.html.feedback.thanks()))
+      Future.successful(Ok(feedbackThankYouView()))
   }
 
   private def CreateQuestionnaireAudit(survey: Questionnaire): Map[String, String] = {
     Map(
-      "easyToUse" -> survey.easyToUse.mkString,
-      "satisfactionLevel" -> survey.satisfactionLevel.mkString,
+      "easyToUse" -> survey.easyToUse.toString,
+      "satisfactionLevel" -> survey.satisfactionLevel.toString,
       "whyGiveThisRating" -> survey.whyGiveThisRating.mkString
```

Comment:
Can this be .toString like the others as well? What was the reason to change from .mkString to .toString?

Reply:
`.mkString` works with collections and iterators. The type was previously Option[Int] which is treated as a collection. `easyToUse` and `satisfactionLevel` are now primitive type Int so we call `.toString` instead.